### PR TITLE
copied function from ZAM to check for noTemplates array

### PR DESCRIPTION
**Pull request by ocke**
:koala:
copied function from ZAM to check for noTemplates array
Describe your PR

/cc @zendesk/quokka

### References
 - Jira link: 

### Risks
 - None

### CODE REVIEW

File: spec/package_spec.rb [hunk 1]
```diff
@@ -110,7 +110,7 @@
 ;
 
     var app = ZendeskApps.defineApp(source)
-      .reopenClass({"location":"ticket_sidebar","singleInstall":false})
+      .reopenClass({"location":"ticket_sidebar","noTemplate":false,"singleInstall":false})
```
Comment (sandlerr) [inline spec/package_spec.rb]: why was this needed?

Comment (sandlerr) [inline spec/package_spec.rb]: I think I get it

**Comment by sandlerr** [comment]
:+1:

**Comment by svizzari** [comment]
Finally ;) :+1:

File: lib/zendesk_apps_support/package.rb [hunk 2]
```diff
@@ -72,6 +72,15 @@ def template_files
       files.select { |f| f =~ /^templates\/.*\.hdbs$/ }
     end
 
+    def no_template
+      manifest = manifest_json
+      if manifest['noTemplate'].is_a?(Array)
+        false
+      else
+        !!manifest['noTemplate']
```
Comment (xuebinglee) [inline lib/zendesk_apps_support/package.rb]: What are `!!` for?

File: lib/zendesk_apps_support/package.rb [hunk 3]
```diff
@@ -72,6 +72,15 @@ def template_files
       files.select { |f| f =~ /^templates\/.*\.hdbs$/ }
     end
 
+    def no_template
+      manifest = manifest_json
```
Comment (xuebinglee) [inline lib/zendesk_apps_support/package.rb]: I don't quite get it. Why not use `manifest_json` directly? No change will be made to it in this method as far as I can see.

File: lib/zendesk_apps_support/package.rb (hunk 2 above)
Comment (ocke) [inline lib/zendesk_apps_support/package.rb]: to make sure it returns a boolean

Comment (xuebinglee) [inline lib/zendesk_apps_support/package.rb]: I see. Smart move.
